src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -192,7 +192,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
       <TargetFramework>$(_TFM)</TargetFramework>
       <RuntimeIdentifier>$(RuntimeIdentifier)</RuntimeIdentifier>
-      <RuntimeFrameworkVersion Condition="'$(FX_Version)' != ''">$(FX_Version)</RuntimeFrameworkVersion>
+      <RuntimeFrameworkVersion Condition="'$(RuntimeFrameworkVersion)' != ''">$(RuntimeFrameworkVersion)</RuntimeFrameworkVersion>
```

Comment:
Ah never mind. I see this is CDATA for generating a project. Funky and easy to miss in a diff with only so much context.